### PR TITLE
Issue/6 Login

### LINKED ISSUE #6: Login
# Description

A user has the possibility to log in and log out

### PR DESCRIPTION
**Pull request by msentwickler**
## Related Issue
closes #6 

## Type of change

- [ ] Bug fix
- [x] Feature request
- [ ] Documenting code

## Checklists

- [x] I referenced the according Issue
  - [ ] I created an Issue
  - [x] I put a link in the Section [Related Issue](#related-issue)
- [ ] I tested my code with multiple browsers
- [x] My commit messages are good and understandable

### CODE REVIEW

File: src/components/Header/Header.tsx [hunk 1]
```diff
@@ -22,7 +22,7 @@ class Header extends React.Component<HeaderInterface> {
                         </Link>
                         <div className="button-box-landing">
                             {/* TODO Route zurück auf /login */}
```
Comment (Akida31) [inline src/components/Header/Header.tsx:24]: this TODO is fixed now!?

File: src/components/Login/Login.tsx [hunk 2]
```diff
@@ -1,106 +1,61 @@
...
-                        <button className="pointer google-login" type="button">
-                            <img
-                                className="google-bild"
-                                src="https://www.freepnglogos.com/uploads/google-logo-png/google-logo-png-suite-everything-you-need-know-about-google-newest-0.png"
-                                alt="log in with google"
+                        <div className="actions flex-row-box">
+                            <div className="checkbox flex-row-box">
+                                <input
+                                    type="checkbox"
+                                    onClick={() => loginStore.setIsStayLoggedIn(!loginStore.isStayLoggedIn)}
```
Comment (Akida31) [inline src/components/Login/Login.tsx:42]: rename this method, name to long. Sth like `setLoginStatus`

File: src/i18n/de/resource.json [hunk 3]
```diff
@@ -17,7 +17,14 @@
       "privacy": "Datenschutzerklärung"
     },
     "login": {
-      "login": "ANMELDEN"
+      "forgotPassword": "Passwort vergessen?",
+      "inputs": {
+        "emailLabel": "E-Mail",
+        "passwordLabel": "Passwort"
+      },
+      "label": "Login",
```
Comment (Akida31) [inline src/i18n/de/resource.json:25]: should this be translated?

File: src/i18n/en/resource.json [hunk 4]
```diff
@@ -17,7 +17,14 @@
...
     },
     "login": {
-      "login": "LOGIN"
+      "forgotPassword": "Forgot password?",
+      "inputs": {
+        "emailLabel": "E-Mail",
+        "passwordLabel": "Password"
+      },
+      "label": "Login",
+      "login": "Login",
```
Comment (Akida31) [inline src/i18n/en/resource.json:26]: duplicated?

File: src/i18n/en/resource.json [hunk 5]
```diff
@@ -17,7 +17,14 @@
...
+      "stayLoggedIn": "Stay LoggedIn"
```
Comment (Akida31) [inline src/i18n/en/resource.json:27]: stay signed in / stay logged in

File: src/pages/Landing/LandingPage.tsx [hunk 6]
```diff
@@ -40,7 +40,8 @@ class LandingPage extends React.Component {
                         <Modal
                             visible
                             closemodal={() => {
-                                // TODO redirect to '/'
+                                // @ts-ignore
+                                this.props.history.push('/');
```
Comment (Akida31) [inline src/pages/Landing/LandingPage.tsx:44]: why this `ts-ignore`? Add a TODO probably

File: src/store/LoginStore.ts [hunk 7]
```diff
@@ -0,0 +1,30 @@
+import {observable, action} from "mobx";
+
+class LoginStore {
+    @observable email = '';
+    @observable password = '';
+    @observable isStayLoggedIn = false;
```
Comment (Akida31) [inline src/store/LoginStore.ts:6]: rename this, too long

File: src/store/LoginStore.ts [hunk 8]
```diff
@@ -0,0 +1,30 @@
...
+    @observable errorText: string = '';
+    @observable valid = false;
+
+    @action setEmail(value: string) {
+        this.email = value;
+    }
+    @action setPassword(value: string) {
+        this.password = value;
+    }
+    @action setIsStayLoggedIn(value: boolean) {
```
Comment (Akida31) [inline src/store/LoginStore.ts:16]: rename this, too long

File: src/i18n/de/resource.json (hunk 3 above)
Comment (msentwickler) [reply src/i18n/de/resource.json:25]: In the design not, so I don't know.

File: src/i18n/en/resource.json (hunk 4 above)
Comment (msentwickler) [reply src/i18n/en/resource.json:26]: No, see german one

**Pull request merged**